felix/bpf-gpl/Makefile

```diff
@@ -65,6 +68,19 @@ C_FILES:=tc_preamble.c tc.c connect_balancer.c connect_balancer_v46.c connect_ba
 all: $(OBJS)
 ut-objs: $(UT_OBJS)
 
+libbpf: $(LIBBPF_FILE_CREATED)
+$(LIBBPF_FILE_CREATED):	
+ifeq ("$(wildcard ./$(LIBBPF_DIR))", "")
+	$(info "Directory does not exist.")
+	@echo "Directory does not exist."
+	git clone --depth 1 --single-branch https://github.com/libbpf/libbpf.git
+endif
+	rm -rf .libbpf-*
+	cd $(LIBBPF_DIR) && \
+		git fetch --tags && git checkout $(LIBBPF_VERSION)
```

Comment:
Just a thought, could you do 
```
git checkout $(LIBBPF_VERSION) ||
  git fetch --tags && git checkout $(LIBBPF_VERSION)
```
If the release is already available locally, that'd save the git fetch.